### ISSUE TITLE
Going Forward: Thoughts from a new user and possible contributor

### ISSUE THREAD

**Issue by davterra** [mynodebtc/mynode#314]
Hi,

I am have been using mynode for 2-3 weeks, the past 7-10 days as a "premium" user.
I run mynode on a virtualbox vm, hosted on a laptop running ubuntu 18.04.

I have several observations and suggestions regarding my experience so far:

My overall observation is that the myNode package is a fantastic concept and very appealing
both functionally, and GUI-wise. There's nothing else out there that does all that myNode 
attempts to do. I've started with my own bitcoind install, and then added dojo, whirlpool, and LND and just that level of integration is not easy....and my setup doesnt even have a comprehensive GUI. So, I congratulate the myNode devs for what they have built so far.

Sadly, after tweaking my installation for 2-3 weeks and observing other's experience on Telegram,
I can't say that I'm satisfied with the execution of the myNode concept. At least, speaking as a "premium" user. To offer something as comprehensive and powerful as myNode to the community for free is a fantastic thing. To charge $100 for something that, for now, seems very fragile and perhaps "not quite ready for prime-time" is a different matter. Compounding my feelings on this is my observation after a week that the "premium" Telegram help channel seems to be useless and virtually ignored.

Having said all of that, I intend on persisting with my efforts to get my installation working, and if I can do that, I am even interested in helping in development or documentation. I have made quite a few contributions to the Wasabi wallet documentation website, and I see a big need here. 
I am a power user of Wasabi and Virtualbox, and could help immediately in those areas of creating "guides". I also have some python and shell-scripting experience, but nothing special there...

Bottom line for me is, I like the myNode concept, and I think it can go a long way in fostering increased adaptation as well as privacy for bitcoin use. That's a mission that I would like to be a part of. But, for me to feel like I'm not wasting my time on a "fragile" product, I would very much like to see a slowdown in adding new features, and instead see that time spent on debugging and helping
users with problems.  The best thing you can say about an app is, "It just works", and that cannot yet be said about myNode. If you can get myNode to the "it just works" stage, you have the potential to be the dominant, go-to app for bitcoiners. Word-of-mouth will make or break you. "It just works" will "make" you....."I have this or that problem with it that I can't solve or get help with" will break you.

I want to see you succeed. 
Just my 2 satoshi's worth......

**Comment by davterra** [comment]
I think its only fair for me to provide an update to what could be interpreted as a negative "review".

After 2 months of constant use, upgrading. testing and experimenting, I must report that my user
experience is much better than what I wrote earlier. I did purchase the premium version, and I am quite happy with the value
of what I received for $100. I run myNode on a Virtualbox vm , on a host running Lubuntu, with only 4gb ram. 
Assigning 3gb to myNode and 1gb to the host has worked fine for me.

I still am of the opinion that new features are perhaps added too quickly at the expense of time spent debugging current issues and providing premium support, but having said that, I am seeing a sharp dropoff in current issues, and the support on the premium channel is noticeably better.

Bottom line: 
I am very, very impressed by myNode overall. Cobbling together all of these tools, and putting them under a very nice GUI is not at all an easy thing to do. 
I am fairly busy, but I as I gain experience with myNode I am trying to help people on the two Telegram channels, and I have a couple of potential PR's in mind that are WIP.

Also, I am better at writing than coding and I would like to perhaps contribute to the myNode Guides, but that material seems to live somewhere other than the github/mynodebtc repos. 

If that's an area where contributions are welcome, please let me know what the procedure is.

**Comment by abhiShandy** [comment]
I agree that this project hasn't reached its full potential but things are improving. Taylor is the only contributor working regularly on this project and he also handles shipping, email queries, telegram queries. He got some help in managing Telegram with few enthusiastic community members (like Amiga500) and probably few more over the summer to answer queries of premium users. But, I don't know why this project hasn't attracted more developers. This project definitely needs lot of work (for example: testing, usage and developer documentation, security audits, debugging guides) and Taylor is overloaded. Perhaps we need a well-written **contribution guide** to attract them.

Regarding general documentation, few guides are available on Github [wiki](https://github.com/mynodebtc/mynode/wiki) and the main website. We would love to have contributors like you adding more guides. Personally, I would prefer to have **Troubleshooting** guides for common issues (for example, looking for drive, Bitcoin is red, Lightning is waiting) and start redirecting Telegram users to that.
